geode-for-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/commands/executor/GeodeRedisServerStartupUsingGfshAcceptanceTest.java

```diff
@@ -149,13 +151,26 @@ public void gfshStartsRedisServer_whenCustomPortAndBindAddress() {
   }
 
   @Test
-  public void gfshDoesNotStartRedisServer_whenNotRedisEnabled() {
+  public void gfshStartsRedisServer_whenAllBindAddress() {
```

Comment:
This test name seems incomplete.

Reply:
Updated to `gfshStartsRedisServer_withAnyLocalBindAddress`.